srcs/runners_manager/vm_creation/aws/AwsManager.py

```diff
@@ -27,7 +29,8 @@ def __init__(
         super(AwsManager, self).__init__(
             name, settings, redhat_username, redhat_password, ssh_keys
         )
-        self.ec2 = boto3.client("ec2")
+        self.tags = settings.get("tags")
+        self.ec2 = boto3.client("ec2", region_name='us-west-2')
```

Comment:
We were using the `AWS_DEFAULT_REGION` environment var to set this setting. What happened?

We cannot hardcode the value in the code.
```suggestion
        self.ec2 = boto3.client("ec2")
```

Reply:
Damn I forgot to remove it or to do it properly, but indeed, I don't know why but if you don't give this parameter in addition it returns an error. I'll look for a way to fix it (or make it cleaner)

Reply:
Here is the error : `botocore.exceptions.NoRegionError: You must specify a region.`

Reply:
Ok, it's good now